lib/internal/url.js

```diff
@@ -19,6 +19,14 @@ const {
   ERR_INVALID_URL_SCHEME,
   ERR_MISSING_ARGS
 } = require('internal/errors').codes;
+const {
+  CHAR_PERCENT,
+  CHAR_PLUS,
+  CHAR_AMPERSAND,
+  CHAR_EQUAL,
+  CHAR_LOWERCASE_A,
+  CHAR_LOWERCASE_Z,
```

Comment:
is the trailing comma required?

Reply:
@gireeshpunathil no, it's not required, but allowed. I can delete it (should I?), but I think it's a better way to work with multiline "lists"

Reply:
I prefer the trailing comma but others don't, I don't think we have consensus, so I'd say do whatever you prefer @daynin .